library/QATools/QATools/PageObject/Page.php

```
@@ -106,4 +113,38 @@ public function setUrlBuilder(IBuilder $url_builder)
 		return $this;
 	}
 
+	/**
+	 * Sets the url matches used for opened check.
+	 *
+	 * @param array $url_match The match for the url.
+	 *
+	 * @return $this
+	 */
+	public function setUrlMatches($url_matches)
+	{
+		$this->urlMatches = (array)$url_matches;
+
+		return $this;
+	}
+
+	/**
+	 * Checks if the page is opened based on the specified url match.
+	 *
+	 * @return bool
+	 */
+	public function opened()
+	{
+		if ( empty($this->urlMatches) ) {
+			return true;
```

A full url match might be the `page-url` annotation? 

Probably its better to throw an exception, cause there is nothing to match against specified.